expression/chunk_executor.go

```diff
@@ -263,3 +260,28 @@ func VectorizedFilter(ctx sessionctx.Context, filters []Expression, iterator *ch
 	}
 	return selected, nil
 }
+
+// VectorizedFilterOneRow applies a list of filters to a row.
+func VectorizedFilterOneRow(ctx sessionctx.Context, filters []Expression, row chunk.Row) (bool, error) {
+	selected := true
+	for _, filter := range filters {
+		isTypeInt := filter.GetType().EvalType() == types.ETInt
+		if isTypeInt {
```

Comment:
Can line 269~284 be extracted as a function?
Thus VectorizedFilter can reuse it.

Reply:
No, if we reuse line 269~284 in the function `VectorizedFilter`, it's execution logic is not **vectorized** any more.

Reply:
Maybe wen can extract line 271~285 to a function named `evaluateOneFilter()` and reuse this function both in `VectorizedFilter` and `FilterRow`.